src/node/desktop/src/main/utils.ts

```diff
@@ -600,3 +603,36 @@ export function getNumericEnvVar(envVarName: string): number | undefined {
   }
   return undefined;
 }
+
+const TRAILING_SLASH_REGEX = /[\\/]+$/;
+
+/**
+ * Removes trailing slashes from a string, optionally keeping the starting slash
+ * or double-slash for UNC paths.
+ * @param str The string to remove trailing slashes from
+ * @param keepStartSlash Whether to keep the starting slash if it exists
+ * @returns The string with trailing slashes removed
+ */
+export function removeTrailingSlashes(str: string, keepStartSlash = false): string {
+  const trailingSlashes = str.match(TRAILING_SLASH_REGEX);
```

Comment:
Could we simplify some of the logic below by detecting the "only slashes" case first? E.g.

```
if (/^[\\/]+$/.test(str)) {
  return str;
}
```

That would remove the need to detect single slashes / UNC paths below.